pzsd_bot/constants.py

```diff
@@ -0,0 +1,51 @@
+from enum import Enum
+
+from pydantic_settings import BaseSettings
+
+
+class EnvConfig(BaseSettings, env_file=".env", extra="ignore"):
+    """Default config to pull from .env file."""
```

Comment:
are these really constant then tho

Reply:
haha, well. True, not everything in it is a constant, I just named the module that. It's a place for constants and settings. Do you think it's confusing or are you just being cheeky?

Reply:
settings.py

Reply:
Fair enough, I'll rename